PRASCapacityCredits.jl/Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "PRASCapacityCredits"
+uuid = "2e1a2ed5-e89d-4cd3-bc86-c0e88a73d3a3"
+authors = ["Gord Stephen <gord.stephen@nrel.gov>"]
+version = "0.7.0"
+
+[deps]
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+PRASCore = "c5c32b99-e7c3-4530-a685-6f76e19f7fe2"
+
+[compat]
+Distributions = "0.25"
+PRASCore = "0.7"
+julia = "1"
```

Comment:
Same Q about Julia requirement, I think it is 1.10 everywhere else

Reply:
Not all of these packages strictly require 1.10, but since PRASCore does it's kind of a moot point anyways - I can set them all to 1.10 for consistency

Reply:
Gotcha, thanks!